project/dexter-monitor/routes/project.js

```diff
@@ -71,6 +75,68 @@ exports.saveSnapshotSummary = function() {
         })
 };
 
+exports.createProject = function(req, res) {
+
+	Promise.resolve(req.body.project)
+	.then(createDexterServerDatabase)
+	.then(insertDexterMonitorEntry)
+	.then((project) => {
+		startDexterServer(project);
+		
+		//wait for the dexter server to start
+		setTimeout( () => {
+			log.info('Dexter Server instance created at:' + project.hostName + ":" + project.portNumber );
+			res.send({status:"ok"});
+		}, 5000);
+    })
+	 .catch((err) => {
+        log.error(`Failed to create Dexter Server instance : ${err.message}`);
+        res.send({status:"fail", errorMessage: err.message});
+    });
+}
+
+function createDexterServerDatabase(project) {
+	var sql = `CREATE DATABASE ${project.projectName}; USE ${project.projectName};`;
+		sql += fs.readFileSync(`${global.config.dexterServerPath}/config/ddl.sql`).toString();	
+		sql += `USE ${global.config.database.name};`;
+	return database.exec(sql).then(() => project);
+}
+
+function insertDexterMonitorEntry(project) {
+	var sql = `INSERT INTO ProjectInfo (projectName, pid, requester, administrator, portNumber, createdDateTime, language, hostIP, projectYear, emailingWhenServerDead, groupName, dbName) VALUES ('${project.projectName}', '${project.projectName}','${project.adminName}', '${project.adminName}', '${project.portNumber}', now(), '${project.language}', '${project.hostName}', YEAR(now()), 'Y', '','${project.projectName}')`;
```

Comment:
In order to avoid SQL Injection attacks, you should always escape any user provided data before using it inside a SQL query. 
You can using the ${mysql.escape(project.projectName)}
https://github.com/mysqljs/mysql#escaping-query-values